NotadogApi.Tests/Hubs/GameHubTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+using NotadogApi.Domain.Game;
+using NotadogApi.Domain.Game.States;
+using NotadogApi.Domain.Users.Models;
+using NotadogApi.Hubs;
+using NotadogApi.Infrastructure;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace NotadogApi.Tests.Hubs
+{
+	class GameHubTests
+	{
+		#region Fields
+
+		private GameHub _gameHub;
+		
```

Comment:
do we need empty rows between class vars?

Reply:
No, we don't. But why shouldn't i add them? We don't have conventions.

Reply:
I think we need to discuss it